### PR TITLE
basic ironic chart

### PR DESCRIPTION
**Pull request by maris-accenture**
**What is the purpose of this pull request?**: Addition of ironic chart 

**What issue does this pull request address?**: Feature #191

**Notes for reviewers to consider**: Charts prepared for review

**Specific reviewers for pull request**: @v1k0d3n

### CODE REVIEW

**Comment by maris-accenture** [comment]
@v1k0d3n  ironic charts converted from sapcc/openstack-helm.

File: ironic/bin/ironic-inspector-start [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -e
+
+. /container.init/common.sh
+
+function process_config {
+    apply_patches # Apply hot-fixes, which need to be either upstreamed or moved to the container build
+
+    cp /ironic-etc/ironic-inspector.conf  /etc/ironic-inspector/ironic.conf
```
Comment (intlabs) [inline ironic/bin/ironic-inspector-start]: If possible could we try and avoid this sort of config manipulation - and load config files directly to their desired location.

File: ironic/bin/ironic-pxe-start [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+. /container.init/common.sh
+
+function process_config {
+    cp /ironic-etc/ironic.conf  /etc/ironic/ironic.conf
```
Comment (intlabs) [inline ironic/bin/ironic-pxe-start]: as above - ideally we shouldn't need to be performing this sort of manipulation on container start.

**Comment by v1k0d3n** [comment]
@maris-accenture thank you for trying to get this in! we're going to close this out and try to start over on ironic once we're in openstack proper.

**Pull request closed without merging**